assembl/static2/js/app/components/common/displayResponseAuthor.jsx

```diff
@@ -0,0 +1,39 @@
+// @flow
+import React from 'react';
+
+export type displayResponseAuthorProps = {
+  authorFullname: string,
+  /** Circle avatar props */
+  parentPostAuthorFullname: ?string,
+  /** Comment displayed published date */
+  displayedPublishedDate: string,
+  /** Comment published date */
+  publishedDate: string,
+  displayIsEdited: any
```

Comment:
shouldn't it be a boolean ?

Reply:
doesn't seem like it... what is this any ?

Reply:
I put any to resolve flow issues and open the PR but I don't know what is the type. displayIsEdited is the small span next to the username that said that the post was modified or not.